src/components/CreateInstructorPage/AreasOfExpertise.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+
+import RenderInputTextField from '../RenderInputTextField';
+import RemoveButton from '../../components/RemoveButton';
+
+
+class AreasOfExpertise extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleRemove = this.handleRemove.bind(this);
+  }
+
+  handleRemove(index) {
+    this.props.fields.remove(index);
+  }
+
+  render() {
+    const { fields } = this.props;
+
+    return (
+      <div className="areas-of-expertise my-4">
+        <button
+          type="button"
+          className="btn btn-light js-add-button"
+          onClick={() => fields.push({})}
+        >
+          Add area of expertise
+        </button>
+        <ul className="list-group p-0 m-0">
+          {fields.map((expertise, index) => (
```

Comment:
The initial part of this and social links are very similar because they are both using `FieldArray` to make a list that adds and removes fields.  I'm happy to refactor it if that's desired, but the refactor was not super intuitive since you need to pass a child component and clone it to pass props to that